### PR TITLE
Implement Navigation and Download App

### PR DESCRIPTION
**Pull request by guillaumegaluz**
screen shot 2018-09-24 at 3 54 14 pm

### CODE REVIEW

**Comment by guillaumegaluz** [comment]
@Jauny for now i'll add the lib's css file until we can load it via webpack

File: components/button.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import React from "react"
```
Comment (Jauny) [inline components/button.tsx]: I don't think you need to import React (Nextjs does it automatically). This is at least the case for pages and I think it's true for all components.

File: components/head.tsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import Head from 'next/head'
```
Comment (Jauny) [inline components/head.tsx]: there is a better way to do this (I might get to it before you), it's overriding the `[pages/_document.tsx](https://nextjs.org/docs/#custom-document)`

File: pages/index.tsx [hunk 3]
```diff
@@ -1,16 +1,20 @@
...
-class Tamere extends React.Component<IProps, object> {
-
-}
\ No newline at end of file
+export default () => (
+  <Layout title='About us'>
+    <GetTheApp title="Buy and sell authentic streetwear and sneakers on your phone." />
+    <img src="static/supreme-box-logo-stack.png"
+       srcSet="static/supreme-box-logo-stack@2x.png 2x,
+               static/supreme-box-logo-stack@3x.png 3x"
```
Comment (Jauny) [inline pages/index.tsx]: 🤩

**Review by Jauny — APPROVED** [review]
No typescript yet?

I'll spend today to see if I can make style-jsx work with sass+typescript and storybook + typescript.

If I can't, I think we can just make the call to revert back to normal javascript (and maybe Flow if we feel like it's necessary).

**Pull request closed without merging**